### DISCUSSION TITLE
Broken map/storyboard

### DISCUSSION THREAD

**Discussion by erensutcu** [ppy/osu#20764] [Q&A] [answered]
I get really low fps and high ms when there is a animation in [this](https://osu.ppy.sh/beatmapsets/448818#osu/1002613) map(check the fps at righ bottom. Even sometimes lazer just get crazy when i try to play this map with storyboard(video 1).  The loading screen takes too much time to load when i play first time after opened lazer(video 5). I tried a lot of things to solve it but no one work(changing drive, redownloading etc.)

Without storyboard, there is no problem.

This is only happens in this map. In other maps storyboards can make a little high ms or a little fps drops but none of them bad as this map. Most of time other maps are really playable. Especially newer storyboards usually doesn't even make a little high ms.

https://user-images.githubusercontent.com/114931666/195833109-06627987-59dd-40e6-83a4-66eb0a2284f3.mp4

https://user-images.githubusercontent.com/114931666/195833601-6a25a216-5133-42c2-b5cb-a74f7737ffde.mp4

https://user-images.githubusercontent.com/114931666/195833653-d745470e-09a4-4533-9d5c-afee07ed54c7.mp4

https://user-images.githubusercontent.com/114931666/195833903-08b00aa5-c94d-44a0-af52-95ad8f0b22ec.mp4

https://user-images.githubusercontent.com/114931666/195835204-c176addc-6775-462a-a22b-9df39ca59a59.mp4

**Comment by peppy** [comment, marked as answer]
Going to be another case of https://github.com/ppy/osu/issues/4976 / https://github.com/ppy/osu/issues/3756. More recently tracked via https://github.com/ppy/osu/issues/12401.
